### PR TITLE
Add alignment for latest posts block

### PR DESCRIPTION
**Pull request by westonruter**
Adds alignment controls:

image

Also fixes margin/padding of Latest Posts block in editor and frontend.

Replaces #1183.
See #1011.

### CODE REVIEW

**Comment by iseulde** [comment]
Could we add the alignment on the block itself for consistency? cc @jasmussen

**Comment by jasmussen** [comment]
> Could we add the alignment on the block itself for consistency?

Yes I think we should, especially since it doesn't have much of a toolbar to begin with.

**Comment by westonruter** [comment]
@jasmussen ok, so then should the widget block mocks be updated to always have the alignment toolbar with the block instead of the inspector? /cc @melchoyce

File: lib/blocks/latest-posts.php [hunk 1]
```diff
@@ -43,8 +48,10 @@ function gutenberg_render_block_core_latest_posts( $attributes ) {
 		$posts_content .= "<li><a href='{$post_permalink}'>{$post_title}</a></li>\n";
 	}
 
+	$class = 'blocks-latest-posts ' . esc_attr( 'align' . $align );
```
Comment (youknowriad) [inline lib/blocks/latest-posts.php]: We should follow the same convention as the frontend blocks to generate the classname. This should be `wp-block-latest-posts`

**Comment by jasmussen** [comment]
Looks, works great! 👍 👍

**Comment by westonruter** [comment]
So ✅ to merge?

**Comment by jasmussen** [comment]
Yup!

**Pull request merged**